Formula/phoneinfoga.rb

```diff
@@ -0,0 +1,39 @@
+class Phoneinfoga < Formula
+  desc "Information gathering framework for phone numbers"
+  homepage "https://sundowndev.github.io/phoneinfoga/"
```

Comment:
Is the software extremely popular? Because we generally discourage people from submitting their own software.

Reply:
Download stats:
![image](https://user-images.githubusercontent.com/16480203/218744993-a279163f-5311-4e4b-b8ba-1580795aaa5b.png)

I see a bunch of software included in this repo have been accepted with much lower stats.

I created this PR in response to [an user having issues](https://github.com/sundowndev/phoneinfoga/issues/1219) installing the software from GitHub binaries.

Reply:
> I see a bunch of software included in this repo have been accepted with much lower stats.

Most software here isn't submitted by the author though.

Reply:
> we generally discourage people from submitting their own software

May I ask why ? I'm fine with letting the community adding it, but it'll be updated by my me at each release anyway.